### PR TITLE
azure: use common makefile for generating images

### PR DESCRIPTION
**Pull request by knrt10**
fixes: #535
Signed-off-by: Kautilya Tripathi <ktripathi@microsoft.com>

### CODE REVIEW

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @knrt10

File: azure/image/Makefile [hunk 1]
```diff
@@ -2,65 +2,11 @@
...
-PAUSE_REPO    = docker://k8s.gcr.io/pause
-PAUSE_VERSION = 3.6
-PAUSE_BUNDLE  = pause_bundle
-
-# Static libseccomp is necessary for kata-agent
-# https://github.com/kata-containers/kata-containers/issues/5044#issuecomment-1239773921
-STATIC_LIB_BUILDER = ../../../kata-containers/ci/install_libseccomp.sh
-STATIC_LIB_DIR     = $(abspath staticlib)
-STATIC_LIB         = $(STATIC_LIB_DIR)/kata-libseccomp/lib/libseccomp.a
+.PHONY: build clean
```
Comment (jtumber-ibm) [inline azure/image/Makefile:7]: Should be `image` not `build` now?

**Review by yoheiueda — APPROVED** [review]
LGTM

**Review by jtumber-ibm — APPROVED** [review]
lgtm

**Pull request merged**